packages/melange-moment/melange-moment.0.3.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Melange bindings for Moment.js"
+maintainer: "Ahrefs"
+authors: "Jiayu Liu <jimexist@gmail.com>"
+license: "MIT"
+tags: ["melange" "org:ahrefs"]
+homepage: "https://github.com/ahrefs/melange-moment"
+bug-reports: "https://github.com/ahrefs/melange-moment"
+depends: [
+  "dune" {>= "3.9"}
+  "ocaml"
+  "melange" {>= "2.0.0"}
+  "reason"
+  "melange-jest" {with-test}
+  "opam-check-npm-deps" {with-test}
```

Comment:
`with-dev-setup` is used in melange-numeral and reason-react; but `with-test` is used in this package as well as melange-test. Is this intentional? What's the difference?

Reply:
I started using `with-dev-setup` as a way to make the packages more compatible with opam 2.2. But over time i have realized it will still take quite some time for that version to be used across the board, so I am rolling back to `with-test` for now.

The difference is that `with-dev-setup` won't install the deps with opam 2.1.

Reply:
Once opam 2.2 becomes the main version used, I will go through the packages that depend on `opam-check-npm-deps` and update them.

Reply:
Thanks for the clarification. It's clear to me now!